MapboxCoreNavigationTests/Heading.swift

```diff
@@ -0,0 +1,35 @@
+import CoreLocation
+
+class Heading: CLHeading {
+	
+	private var _heading: CLLocationDirection
```

Comment:
Please apply the naming style of the codebase (without the underscore).

Reply:
In this case, it is needed as this variable would clash with the super class. It is a workaround so we can mock this class for testing.